forms/chapter-organizer-form.md

```diff
@@ -0,0 +1,70 @@
+# PyLadies Chapter Organizer Registration Form
+
+Please fill in this form if you are:
+
+- Currently a PyLadies chapter organizer
+- Was a chapter organizer and wants to step down
+- A new organizer for an existing chapter.
+
+Please share this form with your co-organizers, as well as please remember
+to fill this form again with any changes in your chapter (i.e. someone
+new is joining the team or someone is stepping down).
+
+We may request you to fill this form annually, to keep our records up to date.
```

Comment:
A rolling annual deadline might be more challenging for people to remember then giving an annual audit date. Recommend that we add "..annually, no later than 31 January,.." (date + month up for consensus).

Reply:
ok that sounds like a good idea. I'll have to first think if we have ways to automate this (sending the form once a year). Is it ok if we leave it as is for now, and once we figure out how to automate, we'll set the annual deadline?

Reply:
I love that idea @Mariatta !